src/lib/components/css/select.css

```diff
@@ -0,0 +1,3 @@
+select::after {
+	content: '▼';
+}
```

Comment:
Is this needed? I don't think it's loaded anywhere.

Reply:
just an unneeded artifact